e2e/specs/planner.e2e-spec.ts

```diff
@@ -113,15 +113,15 @@ describe('Planner, check newly created sessions existence', () => {
 
         it('There should be at least two Session allocated to Second judge - prepare', async () => {
             resourceId = await plannerPage.getResourceIdByName(Judges.AMY_WESSOME);
-
-            const foundedEvents = plannerPage.getAllEventsForTheResource(resourceId).count();
+            const row = await plannerPage.getAllEventsForTheResource(resourceId)
+            const foundedEvents = row.length;
```

Comment:
foundEvents? :) I know you weren't the one to name this const, just taking note ;)